### PR TITLE
Add the major version number to the url

### PR DESCRIPTION
**Pull request by pixelastic**
This will add the instantsearch major version number to the url, under the `is_v` name (for `i`nstant`s`earch `v`ersion).

We currently do nothing with it yet, but this is for future reconciliation between old url and new instant search version.

The diff about `onPopState` is simply refactoring of moving the function in its own method.

### CODE REVIEW

File: widgets/url-sync.js [hunk 1]
```diff
@@ -98,34 +99,38 @@ class URLSync {
...
+
+    forEach(window.document.querySelectorAll('input'), function(i) {
+      i.blur();
     });
+
+    helper.setState(fullState).search();
+  }
+
+  init(state, helper) {
+    this.urlUtils.onpopstate(this.onPopState.bind(this, helper));
```
Comment (bobylito) [inline widgets/url-sync.js]: What about using `=>` instead of using `.bind()`?

Comment (pixelastic) [inline widgets/url-sync.js]: How would you pass `helper` to `onPopState` with `=>`?

Comment (bobylito) [inline widgets/url-sync.js]: Nevermind :)

**Comment by vvo** [comment]
@pixelastic conflicts

File: widgets/url-sync.js [hunk 2]
```diff
@@ -1,4 +1,5 @@
 var AlgoliaSearchHelper = require('algoliasearch-helper').AlgoliaSearchHelper;
+var versionNumberMajor = require('../lib/version.js').split('.')[0];
```
Comment (vvo) [inline widgets/url-sync.js]: Is this english compliant? `majorVersion`?

Comment (bobylito) [inline widgets/url-sync.js]: majorNumber? majorVersionNumber?

**Comment by bobylito** [comment]
LGTM

File: widgets/url-sync.js (hunk 2 above)
Comment (vvo) [inline widgets/url-sync.js]: @pixelastic?

**Pull request merged**